db/migrate/20250123120534_create_social_profiles.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSocialProfiles < ActiveRecord::Migration[8.0]
+  def change
+    create_table :social_profiles do |t|
+      t.string :value
+      t.integer :provider
```

Comment:
I think we might want to add `null: false` to these columns as well as migrating the `provider` to a string so we can see the actual provider value in the database too: 
```suggestion
      t.string :value, null: false
      t.string :provider, null: false
```

Reply:
Integer gives flexibility if provider names change for some reason, otherwise we will need to update them in db. Anyway, no issue with migrating to string.

Reply:
in general in the code base we use string for enums. 

and define them like that 

```ruby
enum :video_provider, %w[youtube mp4 vimeo scheduled not_published not_recorded parent children].index_by(&:itself)
```